methods_and_recursion/mad_libs.rb

```diff
@@ -0,0 +1,70 @@
+#!/usr/bin/env ruby
+
+# This helps to check whether 'a' or 'an' will be used.
+class String
+  def vowel?
+    vowels = %w(a e i o u A E I O U)
+    if vowels.include?(self)
```

Comment:
Pro Tip:

    if ________
      true
    else
      false
    end

can be rewritten as:

    ___________